cmd/daemon/daemon.go

```diff
@@ -26,19 +26,10 @@ func main() {
 
 	log := ctrl.Log.WithName("Daemon Init")
 	log.Info("Daemon init")
-	config := ctrl.GetConfigOrDie()
```

Comment:
Did we optimize this out?

Reply:
It's passed in as part of the init because. ctrl.GetConfigOrDie() reads the config from the env, and that makes the daemon untestable if we call that directly inside the daemon. By passing the config in, it becomes a more modular design. Dependency inversion.